index.html

```diff
@@ -67,25 +68,9 @@
         </div>
     </main>
     <main id="about">
+        <h4 class="title">A Little Bit About Me</h4>
```

Comment:
As mentioned above, you can probably ditch this 'title' class and just add the styles to h4 instead.